src/OrleansLoggerProvider/OrleansLoggerProvider.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
ILogConsumer and all the deprecated logging features will be moved here as much as possible when internal runtime finished migrating off the legacy logging infra